OpenTrons/opentrons-api
# fix(app): Fix gantry homing setting mismatch

Closes RQA-1534


# Overview

This PR fixes "home gantry" setting misalignment between the app and ODD, causing unpredictable gantry homing behavior on restart. Supporting text better clarifies the behavior of the setting. 
### ODD
![Screenshot 2023-09-12 at 2 49 04 PM](https://github.com/Opentrons/opentrons/assets/64858653/cb0f82b5-57ca-462d-99ce-37013843e5f8)
### App
![Screenshot 2023-09-12 at 2 50 32 PM](https://github.com/Opentrons/opentrons/assets/64858653/82a4386e-44fa-432e-9e61-fa502200095d)





# Test Plan

- Verify updated buttons.
- Restart the robot when the gantry is not in "home" position. Observe the z-axis reset upon reboot. (Easy way to test this is to start a remove pipette instrument flow (moving the gantry), then restart the robot through the app). 



# Changelog

- Updated gantry button text and behavior to make it more in-line with other robot settings.



# Review requests



- Test on physical robot

# Risk assessment


low
